### PR TITLE
Better drug/drugdelta seeds!

### PR DESCRIPTION
**Pull request by erictzeng**

### CODE REVIEW

**Comment by erictzeng** [comment]
This makes five drugs, but if we want I have a python script that can generate more for us.

File: db/seed/pharmacy/data/sutryn.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-name: Drug2
+name: Sutryn
```
Comment (danwang) [inline db/seed/pharmacy/data/sutryn.yaml:2]: Spelling? Did you mean `Surtyn`?

**Comment by danwang** [comment]
Otherwise, it looks okay

File: db/seed/pharmacy/data/sutryn.yaml (hunk 1 above)
Comment (erictzeng) [inline db/seed/pharmacy/data/sutryn.yaml:2]: i hate you

**Pull request merged**